cmf-cli/resources/template_feed/data/Data.Package/cmfpackage.json

```diff
@@ -57,9 +57,16 @@
       "contentType": "ProcessRulesPre"
     },
     {
-      "source": "MasterData/$(version)/*",
+      "source": "MasterData/App/$(version)/*",
```

Comment:
This has no meaning for customization, I'd rather keep this in an if for and the old structure as default